src/components/NewsFeed.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import { onChildAdded, ref as databaseRef, push, set } from "firebase/database";
+import { database } from "../firebase";
+import "./App.css";
+import Card from "react-bootstrap/Card";
+import { AiFillHeart, AiOutlineHeart } from "react-icons/ai";
+import { Button } from "react-bootstrap";
+
+const NewsFeed = () => {
+  const [posts, setPosts] = useState([]);
+  const [toggleLike, setToggleLike] = useState(false);
```

Comment:
I think it would be better to name this like a boolean. "isLiked" or something along these lines, and make it reflect as such in the db as well. For example "post.val.toggle" does not really tell me what kidn of value that is or what it is used for.